recipes/alignoth/build.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash -e
 
-# TODO: Remove the following export when pinning is updated and we use
-#       {{ compiler('rust') }} in the recipe.
-export \
-    CARGO_NET_GIT_FETCH_WITH_CLI=true \
-    CARGO_HOME="${BUILD_PREFIX}/.cargo"
+export CFLAGS="${CFLAGS} -O3 -fno-rtti"
+export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
```

Comment:
I guess this is just the new default? Or should this be explained in some comment?

Reply:
Hope it is - copied it over from a much newer package.

Reply:
Fingers crossed... :swe